lib/Backend/GlobOpt.cpp

```diff
@@ -4663,6 +4789,13 @@ GlobOpt::OptInstr(IR::Instr *&instr, bool* isInstrRemoved)
 
     this->OptArguments(instr);
 
+    //StackArguments Optimization - We bail out if the index is out of range of actuals.
+    if (instr->m_opcode == Js::OpCode::LdElemI_A && instr->DoStackArgsOpt(this->func) && !this->IsLoopPrePass())
+    {
+        AssertMsg(!instr->HasBailOutInfo(), "There should not be any existing bail out on LdElem instr, at this time.");
```

Comment:
Is there anything functionally wrong if it does have a bailout? In other words, what does this assert protect against?